source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Introducing Minishift - Run OpenShift locally
+date: 2017-03-30 05:45 UTC
+author: budhram
+published: true
+comments: true
+tags: minishift, atomic-developer-bundle, openshift-local
+---
+
+[Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle) has been a great tool so far in providing a prepackaged development environment filled with production-grade, pre-configured tools, that makes the container developer's lives easier.
+
+However, many of our users experienced that the downloads of the vagrant box `projectatomic/adb` of around 500 MB sometimes take time and even fails to download in a slow networks. This and few other potential issues which I would like to address in my another blog post impelled us to think about the better user experience.
+
+From the past few months, we have been looking and investigating for better solutions which could give user a better experience.
+
```

Comment:
IMO this is also extra detail, which an user will not be interested much. Rather we should mention that Minishift is a drastic improvement on ADB and we have been working on Minishift from last few months.